docs/web-apps/automated-testing/cypress/yaml/v1.md

```diff
@@ -279,6 +279,10 @@ A property containing one or more environment variables that are global for all
 Since environment variables are provided to Cypress directly, avoid using `CYPRESS_` as a prefix.
 :::
 
+:::caution
+Currently, `env` only supports lowercase keys on `macOS` platform. We'll add enhancement later.
```

Comment:
> We'll add enhancement later.

False promise detected ;) 🛑 

It'd be good to add some explanation/workaround for the user here.

1. We know that `saucectl run --env` works across the board, right?
2. We know that the suite level `env` attribute works fine across the board as well.

Both are working as expected and should be preferred over the root level `env` attribute.